components/base/base.go

```diff
@@ -57,13 +57,8 @@ type Base interface {
 	// linear is in mmPerSec
 	// angular is in degsPerSec
 	SetVelocity(ctx context.Context, linear, angular r3.Vector, extra map[string]interface{}) error
-}
 
-// A LocalBase represents a physical base of a robot that can report the width of itself.
-type LocalBase interface {
```

Comment:
:+1: Thanks for getting rid of `LocalBase` and putting everything into `Base`. I don't see any reason to distinguish local from nonlocal bases.

Reply:
Local is just a way to wrap an interface with extra exported functions, ideally we'd only have exported functions that fulfill our API, but sometimes we need that flexibility, in this case, to expose the base's width to someone trying to external math or planning. 

No longer needed though - so got rid of it!